### PR TITLE
added views

### PR DESCRIPTION
**Pull request by Becky449**
- Implement the design from the sneak peek wireframes.
- Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,2 +1 @@
-<h1>Here your users list</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+<%= render "user" %>
```
Comment (VICTOR-CHUKWU) [inline app/views/users/index.html.erb:1]: - [ ] kindly match the design given, currently images are bigger than image div and also the screen
Screenshot from 2023-05-20 07-15-43

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
+   <% post.recentfivecomments.each do |comment| %>
+      <ul class="comment width"> 
+          <li> <%= comment.author.name %>: <%= comment.text%> </li>
+       </ul>
+       <% end %> 
+   </div>
+   <% end %>
+  </div>
+  <button type:submit class="all-posts">Pagination</buttom>
+</div>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/index.html.erb:33]: - [ ] kindly fix the image as it is too large
Screenshot from 2023-05-20 07-15-20

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,2 +1,2 @@
-<h1>Here your user with id</h1>
-<p>Find me in app/views/users/show.html.erb</p>
+<%= render "userid" %>
```
Comment (VICTOR-CHUKWU) [inline app/views/users/show.html.erb:1]: - [ ] kindly fix the image in the user show page
Screenshot from 2023-05-20 07-21-35

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Becky449  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @Becky449 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**